assets/src/js/block-button.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * JS for Login with Google Block.
+ *
+ * @package login-with-google
+ */
+
+const {registerBlockType} = wp.blocks;
+const {__} = wp.i18n;
+const {InspectorControls, RichText, useBlockProps} = wp.blockEditor;
+const {Panel, PanelBody, CheckboxControl} = wp.components;
+
+/**
+ * Register block type.
+ *
+ * @arg metadata ( object )
+ * @arg settings ( object )
```

Comment:
Instead of `@arg` we need to use `@param`
https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#unsupported-jsdoc-tags